client/blocks/reader-post-options-menu/index.jsx

```diff
@@ -30,7 +29,7 @@ import { isAutomatticTeamMember } from 'reader/lib/teams';
 import { getReaderTeams } from 'state/selectors';
 import ReaderPostOptionsMenuBlogStickers from './blog-stickers';
 import ConversationFollowButton from 'blocks/conversation-follow-button';
-import { shouldShowComments } from 'blocks/comments/helper';
+import { shouldShowConversationFollowButton } from 'blocks/conversation-follow-button/helper';
```

Comment:
FFTI because this is super nitty:

since `shouldShowConversationFollowButton` will _always_ be used in tandem with ConversationFollowButton, and we are nowhere near an issue of having a  "monolith" file, it could be nicer to export that function from `blocks/conversation-follow-button` instead of moving it to a separate helper file.

That would make this line:
```js
import ConversationFollowButton, { shouldShowConversationFollowButton } from 'blocks/conversation-follow-button';
```

Reply:
I understand the reasoning there, but I'm trying to be consistent with similar helper functions in `blocks/comments` (like `shouldShowComments`), which live in a helper.js.